docs/product/security/ai-ml-privacy.mdx

```diff
@@ -0,0 +1,40 @@
+
+We’re asking for access to the following forms of service data:
+
+- Error message
+- Stack trace
+- Span
+- DOM interactions
+
+Our intended improvements will harness text embeddings – converting words into numerical values that capture their semantic information. In this context, we’ll treat embeddings as a set of features for a downstream task. For example, we may use this aggregated dataset to train a model that can predict the severity of a new issue. **The information within these embeddings cannot identify specific organizations, projects, or issues**. Moreover, our policy on GDPR deletion requests will apply to any data we leverage for AI features - if we are required to delete any sensitive data in our production database, we will also delete the corresponding numerical embedding from our vector database.
+
+### Generative AI
+
+A subset of upcoming features will use Generative AI and Retrieval Augmented Generation (RAG). In this context, we will use text embeddings as an index - a way to retrieve the most contextually relevant bits of source code (via our GitHub integration) or events for a particular Sentry issue. When it comes to RAG, embeddings will always be logically separated (never cross customer boundaries). We may use embeddings to provide improved context to our suggested fix feature, help with semantic search, or improve the relevance of the issue details page, but only for the specific customer whose service data those embeddings were derived. All functionality leveraging RAG will require user opt-in; if you do not intend to take advantage of these features, nothing will change.
+
+### Data Handling
+
+In addition to the consent mechanisms mentioned above:
+
+1. We will continue to encourage all our customers to use our [various data scrubbing tools](https://docs.sentry.io/product/data-management-settings/scrubbing/) so that service data is sanitized before we receive it.
+2. We will apply the same deletion and retention rules to our training data as we do to the underlying service data. This means that if you delete service data, it will also be removed from our machine learning models automatically.
+3. We will scrub data for PII before it goes into any training set.
+4. We will ensure that the only service data presented in the output of any ML feature belongs to the customer using the feature.
+5. We will only use AI models built in-house or provided by our existing trusted [third-party sub-processors](https://sentry.io/legal/subprocessors/) who have made contractual commitments that are consistent with the above.
+
+We are confident that with these controls in place, we will be able to use service data and usage data to improve our products through AI while still protecting that very data.
```

Comment:
```suggestion
1. We'll continue to encourage all customers to use our [various data scrubbing tools](https://docs.sentry.io/product/data-management-settings/scrubbing/) so that service data is sanitized before we receive it.
2. We'll apply the same deletion and retention rules to our training data as we do to the underlying service data. This means that if you delete service data, it will also be removed from our machine learning models automatically.
3. We'll scrub data for PII before it goes into any training set.
4. We'll ensure that the only service data presented in the output of any ML feature belongs to the customer using the feature.
5. We'll only use AI models built in-house or provided by our existing trusted [third-party sub-processors](https://sentry.io/legal/subprocessors/) who have made contractual commitments that are consistent with the above.

We're confident that with these controls in place, we'll be able to use service and usage data to improve our products through AI while at the same time protecting that data.
```